.vscode/settings.json

```diff
@@ -15,5 +15,7 @@
   ],
   "workbench.editorAssociations": {
     "*.db": "sqlite-viewer.view"
-  }
+  },
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "typescript.enablePromptUseWorkspaceTsdk": true
```

Comment:
I'm not sure I want to enable this. I'm pretty sure we always want people to use the built-in tsdk if it exists and a prompt would be annoying I think.

Reply:
@kentcdodds This is the only way to enable it due to security, as per [this answer on StackOverflow](https://stackoverflow.com/a/75357623):
<img width="562" alt="Screenshot 2023-12-10 at 12 44 19" src="https://github.com/epicweb-dev/epic-stack/assets/57627858/1b05fa83-2f84-4505-90af-7e418952da76">

**If you don't like this, I understand and feel free to close this PR!**

Reply:
I just realized I already have the `"typescript.tsdk": "node_modules/typescript/lib",` setting enabled in my config, but I don't have `"typescript.enablePromptUseWorkspaceTsdk": true` enabled. So is this saying that my `typescript.tsdk` setting is just not doing anything? 🤔